File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-loop-through-found-items-in-documents_5.cs
        private void FindLoop() 
        { 
            int intFound = 0; 
            Word.Range rng = this.Content; 

            rng.Find.ClearFormatting(); 
            rng.Find.Forward = true; 
            rng.Find.Text = "find me"; 

            rng.Find.Execute(
                ref missing, ref missing, ref missing, ref missing, ref missing, 
                ref missing, ref missing, ref missing, ref missing, ref missing,
                ref missing, ref missing, ref missing, ref missing, ref missing);

            while (rng.Find.Found) 
            { 
                intFound++;
                rng.Find.Execute(
                    ref missing, ref missing, ref missing, ref missing, ref missing, 
                    ref missing, ref missing, ref missing, ref missing, ref missing,
                    ref missing, ref missing, ref missing, ref missing, ref missing);
            } 

            MessageBox.Show("Strings found: " + intFound.ToString()); 
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-loop-through-found-items-in-documents_5.vb
    Private Sub FindLoop()
        Dim intFound As Integer = 0
        Dim rng As Word.Range = Me.Content

        rng.Find.ClearFormatting()
        rng.Find.Forward = True
        rng.Find.Text = "find me"

        rng.Find.Execute()

        Do While rng.Find.Found = True
            intFound += 1
            rng.Find.Execute()
        Loop

        MessageBox.Show("Strings found: " & intFound.ToString())
    End Sub